docs/3.0/resources.md

```diff
@@ -691,3 +692,14 @@ class Avo::Resources::Project < Avo::BaseResource
   self.index_query = -> { query.unscoped }
 end
 ```
+:::
+
+:::option self.countless
```

Comment:
This doesn't tell anything about how it will look like. WE need to say that the page numbers will not be visible, etc.